lib/iso_builder.py

```diff
@@ -156,8 +156,7 @@ def _save(self):
         utils.force_symlink(self.timestamp, latest_dir)
 
         iso_file = "%s-DVD-%s-%s.iso" % (self.distro, self.arch, self.version)
-        checksum_file = ("%s-%s-%s-CHECKSUM" %
-                         (self.distro, self.version, self.arch))
+        checksum_file = "%s-CHECKSUM" % iso_file
```

Comment:
That's better, but this uppercase suffix still looks weird to me. What about appending `.md5` or whatever the hash algorithm is?